internal/oci/container.go

```diff
@@ -441,7 +441,10 @@ func (c *Container) verifyPid() error {
 	}
 
 	if startTime != c.state.InitStartTime {
-		return errors.New("PID running but not the original container. PID wrap may have occurred")
+		return errors.Errorf(
+			"PID %d is running but has start time of %d, whereas the saved start time is %d. PID wrap may have occurred",
```

Comment:
I know this is old, but will people know what "pid wrap" is?

Reply:
a quick google of `pid wrap linux` gives me the answer, so I  don't think there's any more explaination needed